src/argilla/client/apis/datasets.py

```diff
@@ -182,7 +182,7 @@ def scan(
             limit = None
 
         request = {
-            "fields": list(projection) if projection else ["id"],
+            "fields": projection,
```

Comment:
here, we must be sure that `fields` attribute is a list. Also, since is an optional argument, I would keep the original setup.

```suggestion
            "fields": list(projection) if projection else ["id"]
```

Reply:
@frascuchon sure, but ` else ["id"]` is a bug. The validation requirements expect at least id and inputs.

Reply:
Sorry, I missed your answer.

This method will be used in a wider way to fetch part of records from a dataset, without parsing into the Record structure. So, is not really a bug, is the default behavior: retrieve at least the record ids.

The error must be handled from the rb.load functions, where records will be parsed into Record instances, so is there where a minimum value of fields must be assured

For text classification, should be `id` and `inputs`
For token classification: `id`, `text` and `tokens`
For text2text, `id` and `text`